apcd-cms/src/client/src/components/Admin/Extensions/AdminExtensions.tsx

```diff
@@ -1,18 +1,102 @@
-import React, { useEffect, useMemo } from 'react';
-import { useExtensions, ExtensionRow } from 'hooks/admin';
+import React, { useEffect, useState } from 'react';
+import { useExtensions, ExtensionRow, ExtensionEditRow } from 'hooks/admin';
+import LoadingSpinner from 'core-components/LoadingSpinner';
+import Paginator from 'core-components/Paginator';
+import styles from './ExtensionList.module.css';
+import ViewExtensionModal from 'apcd-components/Extensions/ViewExtensionModal/ViewExtensionModal';
+import EditExtensionModal from 'apcd-components/Extensions/EditExtensionModal/EditExtensionModal';
 
 export const AdminExtensions: React.FC = () => {
-  const { data, isLoading, isError } = useExtensions({});
+  const [status, setStatus] = useState('All');
+  const [org, setOrg] = useState('All');
+  const [page, setPage] = useState(1);
+  const { data, isLoading, isError, refetch } = useExtensions(status, org, page);
+  const [isViewModalOpen, setIsViewModalOpen] = useState(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState(false);
 
-  if (isLoading) {
+    const [selectedExtension, setSelectedExtension] =
+    useState<ExtensionRow | null>(null);
+
+  const clearSelections = () => {
+    setStatus('');
+    setOrg('');
+    setPage(1);
+  };
+
+  useEffect(() => {
+    refetch();
+  }, [status, org, page, refetch]);
+
+   if (isLoading) {
```

Comment:
```suggestion
   if (isLoading) {
   <LoadingSpinner/>
```
You'll need to import this from core-components.

Reply:
added to latest commit